### DISCUSSION TITLE
Fly.io Cron Jobs

### DISCUSSION THREAD

**Discussion by JohnMatychuk** [towfiqi/serpbear#145] [Q&A]
I have serpbear up and running on fly.io but for some reason I am only getting keywords to update if I manually log in and update.
Is there something additional that I need to configure to get this to work on fly.io or is that a limitation of the service?
Thanks!

**Comment by rossdav70** [comment]
I'm self hosting on my own server and am seeing the same issue. Cron jobs don't appear to be running.

When I run docker logs [container name] I see errors regarding authentication.

Could really do with understanding how to fix this

**Reply by towfiqi** [reply]
> When I run docker logs [container name] I see errors regarding authentication.

Can you please share the error message?

**Reply by vf65** [reply]
Hi Towfiqi,

Your SerpBear application is fantastic! Unfortunately, I’m experiencing the same issue as JohnMatychuk, and I would like to know if you have a solution to ensure that cron jobs run on fly.io. For context, I deployed the app on a machine with the default configuration, using machine autostop and autostart with min_machines_running = 0.

Thank you so much for your help!
